test/scenario-test.ts

```diff
@@ -54,14 +54,24 @@ export class Process {
         for (const k in extraOption) {
             cmdOption.push('--' + k + '=' + extraOption[k]);
         }
-        const cmd =
+        let cmd =
             path.join(`${__dirname}`, '/../../' + 'node_modules/.bin/autorest') +
             ' --version=3.0.6320 --az --use=' +
```

Comment:
will you also update this version?

Reply:
This is autorest core version. I think we'd better not change it